boilerplate/.maestro/FavoritePodcast.yaml

```diff
@@ -0,0 +1,34 @@
+# flow: run the login flow and then navigate to the demo podcast list screen, favorite a podcast, and then switch the list to only be favorites.
```

Comment:
Should we remove these tests if the user opts out of the demo code?

Reply:
Yeah these should be removed, because then the tests will fail